masschallenge/django-accelerator
# [AC-9385]: Add startupUpdate models

[AC-9385](https://masschallenge.atlassian.net/browse/AC-9385)

How to test
- Ensure that all model definitions on [AC-9385](https://masschallenge.atlassian.net/browse/AC-9385) are met
